twake/frontend/src/app/components/SearchPopup/SearchPopup.js

```diff
@@ -30,7 +30,7 @@ export default class SearchPopup extends React.Component {
       filterType: Search.type || false,
     };
 
-    this.search_enabled = (InitService.server_infos || {}).elastic_search_available !== false;
+    this.search_enabled = true;
```

Comment:
Can you just remove all usage of this variable instead?

Reply:
Done :+1: